ui/partials/perf/alertsctrl.html

```diff
@@ -140,6 +140,15 @@
             <a ng-if="alert.series_signature.has_subtests" href="{{alert.getSubtestsURL(alertSummary)}}" target="_blank" rel="noopener">subtests</a>
           </span>
         </td>
+        <td class="alert-labels">
+          <button class="btn"
+                  ng-class="{'btn-primary visible': alert.significant, 'btn-outline-secondary': !alert.significant}"
+                  ng-disabled="!user.is_staff"
+                  ng-hide="!user.is_staff && !alert.significant"
+                  ng-click="alert.toggleSignificance()">
+            significant
+          </button>
+        </td>
```

Comment:
How about we use a star icon for this (filled if significant, unfilled if not) and put it on the left just after the checkbox, this metaphor will be familiar for people who use gmail and should be a little more space efficient. 

![image](https://user-images.githubusercontent.com/20569/39145525-fe4f7744-4701-11e8-98af-0b5f2ac4876d.png)


font awesome has some star icons you can use for this purpose:

https://fontawesome.com/icons?d=gallery&q=star